### ISSUE TITLE
docs for ivy ɵrenderComponent

### ISSUE THREAD

**Issue by xiaoxiangmoe** [angular/angular#34856]
# 📚 Docs or angular.io bug report

### Description

It would be great if you could tell me where to find the documentation for ivy ɵrenderComponent.

I need ɵrenderComponent for rendering dynamically component created by markdown text.

Can we renderComponent in Angular ivy with extra component/directive declarations?

https://stackoverflow.com/questions/59788660/rendercomponent-in-angular-ivy-with-extra-component-directive-declarations 

Thank you very much.

**Comment by ajitsinghkaler** [comment]
@xiaoxiangmoe api with ɵ on the front are not public API and are bound to change angular does not recommend using these api directly. As these are bound to change hence no documentation

**Comment by pkozlowski-opensource** [comment]
Yep, for now consider this a private API that can change at any moment. As such we don't want to commit / document this API at this point in time.
